### PR TITLE
Issue #5 and #6 write and test global and local installation instructions.

### PR DESCRIPTION
**Pull request by Traap**
This pull-request is to confirm the installation section of GettingStarted.pdf is adequate.
[GettingStarted.pdf](https://github.com/Traap/tlc-article/files/695443/GettingStarted.pdf)

### CODE REVIEW

**Comment by dlewissandy** [comment]
You may want to make sure that the following prerequisites are mentioned in an appropriate location:
* Github authentication relies upon ssh-agent for authentication.   It should be running.
* You must have added your ssh key using the ssh-key command (e.g. ssh-key foo_key).   You can tell if you have a key configured by typing ssh-key -l

**Comment by dlewissandy** [comment]
You may want to mention that the install directories can be looked up using

_kpswhich -var-value TEXMFLOCAL_

or 

_kpswhich -var-value TEXMFHOME_

File: GettingStarted.tex [hunk 1]
```diff
@@ -13,16 +23,43 @@
...
 \subsection{Global installation}
+A global installation is done by installing \tlcA\ into your /path/to/your/texmf
+directory.  Assuming a TexLive installation exists at \texDist\ the following
+shell commands will make \tlcA\ available to your \LaTeX\ environment.
+
+\begin{enumerate}
+  \item cd \tlcHome
+  \item git clone \tlcRepo\
+  \item cd \tlcProduct\
+  \item mkdir \tlcGlobalDist
```
Comment (dlewissandy) [inline GettingStarted.tex]: On linux, you must pass the -p parameter to create a nested subdirectory.

e.g. sudo mkdir -p $(kpsewhich -var-value TEXMFLOCAL)/tex/latex/tlc-article

File: GettingStarted.tex [hunk 2]
```diff
@@ -13,16 +23,43 @@
...
+  \item cp -v \tlcProduct.cls \tlcGlobalDist/.
```
Comment (dlewissandy) [inline GettingStarted.tex]: This command may require elevated permission

File: GettingStarted.tex [hunk 3]
```diff
@@ -13,16 +23,43 @@
...
+  \item sudo mktexlsr \texDist\
+\end{enumerate}
```
Comment (dlewissandy) [inline GettingStarted.tex]: can you add a verification step.   When I type 

find `kpsewhich --var-value TEXMFDIST` --name *.cls  

I do not see the new class listed.

**Review by dlewissandy — CHANGES_REQUESTED** [review]

**Comment by Traap** [comment]
@dlewissandy I addressed each recommendation you made.  I also tested the implementation on my Linux Mint installation.  

[GettingStarted.pdf](https://github.com/Traap/tlc-article/files/699923/GettingStarted.pdf)

**Review by dlewissandy — APPROVED** [review]

**Pull request merged**